_posts/nataliele/2016-02-02-nataliele-chapter5exercise.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+author: nataliele
+title: "Natalie's Chapter 5 exercises"
+---
+
+Even though we havent covered lists, I feel it's impossible to finish the exercise without them so I had to consult stackexchange.
+
+For exercise 1, I got the list and append code from stackexchange. After that it is pretty straightforward.
+<iframe src="https://trinket.io/embed/python/125846c8b7" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+For exercise 2, I quickly figured out it is not as easy as it seems. When I tried a series with a double digit number, the program doesn’t recognize that it's a double-digit number. Then I realized that I have to convert the string of user input into numbers. Which means I have to delimit the user input first and then convert it to number. I got the codes from stackexchange. 
+<iframe src="https://trinket.io/embed/python/4e9dec13ca" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
```

Comment:
In this code, the intention was to prompt for input like in Exercise one, rather than an actual list of numbers.  the line `prompt="Enter a series of number, separated by ',':"` and the `userinput=user.split(delimiter)` are technically correct but don't use loops like the problem intended.